doc/changes/changes_0.2.1.md

```diff
@@ -0,0 +1,9 @@
+# Server for the Exasol Test-Setup Abstraction 0.2.1, released 2022-06-07
+
+Code name: 
```

Comment:
Code name missing. Depends on whether you need a quick release whether you already need one.